lib/moesif_rack/moesif_middleware.rb

```diff
@@ -100,6 +101,26 @@ def @moesif_helpers.log_debug(message)
       end
     end
 
+    def parse_multipart(multipart_form_data, content_type)
```

Comment:
I'd suggest to have a begin/rescue block incase for some reason the parsing doesn't work, we don't want to error out

Reply:
the `parse_body` function which calls this already have begin and rescue, so that will catch this before. so I thought it is not needed here.